### PR TITLE
Bump HiveMQ version from 4.6.0 to 4.20.0

### PR DESCRIPTION
**Pull request by LukasBrand**
https://hivemq.kanbanize.com/ctrl_board/22/cards/16937/details/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,36 +9,42 @@
...
-| file&#x2011;expiration   | 360      |    Timeout in seconds after which the created Blob will be deleted by other nodes, if it was not updated in time.
-| update&#x2011;interval   | 180      |    Interval in seconds in which the Blob will be updated. Must be less than file-expiration.
+| Config Name              |  Default Value   | Description                                                                                                                                                                                                                        |
+|--------------------------|:----------------:|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| connection&#x2011;string |                  | The required connection string of your Azure Storage Account. See the [Azure Documentation](https://docs.microsoft.com/de-de/com.hivemq.extensions.azure/storage/common/storage-configure-connection-string) for more information. |
```
Comment (afalhambra-hivemq) [inline README.md:43]: 404 - for this link.

I think it should be:

```suggestion
| connection&#x2011;string |                  | The required connection string of your Azure Storage Account. See the [Azure Documentation](https://learn.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string) for more information. |
```

File: README.md [hunk 2]
```diff
@@ -9,36 +9,42 @@
...
+HiveMQ instances are added at runtime as soon as they become available by placing their information, on how to connect
+to them, to the configured Azure Storage Container.
 The extension will regularly check the configured Azure Storage Container for files from other HiveMQ nodes.
 Additionally, every broker updates its own file on a regular basis to prevent the file from expiring.
 
 ## Installation
 
-* Download the extension from the [HiveMQ Website](https://www.hivemq.com/releases/extensions/hivemq-azure-cluster-discovery-extension-1.0.0.zip) or from the [GitHub Releases Page](https://github.com/hivemq/hivemq-azure-cluster-discovery-extension/releases/latest).
+* Download the extension from
+  the [HiveMQ Website](https://www.hivemq.com/releases/extensions/hivemq-azure-cluster-discovery-extension-1.0.0.zip) or
```
Comment (afalhambra-hivemq) [inline README.md:23]: I believe latest release is 1.1.0

```suggestion
  the [HiveMQ Website](https://www.hivemq.com/releases/extensions/hivemq-azure-cluster-discovery-extension-1.1.0.zip) or
```

File: arm-quickstart-templates/hivemq-vm-cluster/azuredeploy.json [hunk 3]
```diff
@@ -1,392 +1,380 @@
...
+          "name": "[concat('default/', variables('containerName'))]",
+          "dependsOn": [
+            "[variables('storageAccountName')]"
+          ]
+        }
+      ]
+    },
+    {
+      "type": "Microsoft.Network/publicIPAddresses",
+      "apiVersion": "2020-05-01",
```
Comment (afalhambra-hivemq) [inline arm-quickstart-templates/hivemq-vm-cluster/azuredeploy.json:112]: curious, why this change? previous one was `2015-06-15`

Maybe below property for [`Microsoft.Network/publicIPAddresses`](https://github.com/hivemq/hivemq-azure-cluster-discovery-extension/blob/8fc9ad9f552f75f97e13090e8204eddcf42b06f4/arm-quickstart-templates/hivemq-vm-cluster/azuredeploy.json#L125) should be in sync with this?

Comment (LukasBrand) [reply arm-quickstart-templates/hivemq-vm-cluster/azuredeploy.json:112]: I saw the one being version 2020 and the other being 2015. So I bumped the 2015 one to align both to the same version.

**Review by afalhambra-hivemq — APPROVED** [review]
LGTM
